frontend/components/FieldBoolean/FieldBooleanEdit.jsx

```diff
@@ -83,15 +99,15 @@ export default class FieldBooleanEdit extends Component {
   }
 
   render() {
-    const {onChange, value, schema} = this.props
+    const {displayName, onChange, value, schema} = this.props
 
-    const publishBlock = schema.publish
+    const publishBlock = schema.publish || {}
     const readOnly = publishBlock.readonly === true
 
     return (
       <Label
         compact={true}
-        label={schema.label}
+        label={displayName || ''}
```

Comment:
I see this pattern (`displayName || ''`) repeated across this PR. I believe that is because of [this check](https://github.com/dadi/publish/blob/develop/frontend/components/Label/Label.jsx#L121) in the `Label` component, that stops the component (and its children) from being rendered if the `label` param is falsy.

Instead of supplying a fallback each time we call `Label`, should we add the fallback to the component instead? I'm not sure what the behaviour would be like, since we're talking about rendering a label without any text. Perhaps the logic should be to simply render the children without a wrapping `<label>` element when the `label` prop isn't a String?

Either way, I would feel more comfortable in handling that case in the component itself, rather than ensuring we supply a fallback each time. What do you think?